.github/workflows/release-version-on-merge.yml

```diff
@@ -0,0 +1,27 @@
+name: On Release PR merged
+on:
+  pull_request:
+    types:
+      - closed
+
+permissions:
+  contents: write
+
+jobs:
+  release-version-on-merge:
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.title, '[release:')
```

Comment:
This means the release PR will need to have their title prefixed with `[release:`, right? Is that documented somewhere?